src/wazuh_db/wdb.h

```diff
@@ -1226,6 +1236,14 @@ int wdb_scan_info_fim_checks_control (wdb_t * wdb, const char *last_check);
 // Upgrade agent database to last version
 wdb_t * wdb_upgrade(wdb_t *wdb);
 
+/**
+ * @brief Function to upgrade Global DB to the last version.
```

Comment:
I think `latest` is more appropriate than `last` here. But its correct too.

Reply:
Fixed. Thank you!